modules/task/migrate.go

```diff
@@ -117,6 +117,7 @@ func runMigrateTask(t *models.Task) (err error) {
 		_ = t.UpdateCols("message")
 	})
 	if err == nil {
+		t.Repo = repo
```

Comment:
it might just be better to set this at line 110 above though.

Reply:
done.